docs/related.md

```diff
@@ -29,33 +29,34 @@ or [pull request](https://github.com/helm/helm/pulls).
 - [Technosophos's Helm Plugins](https://github.com/technosophos/helm-plugins) - Plugins for GitHub, Keybase, and GPG
 - [helm-convert](https://github.com/ContainerSolutions/helm-convert) - Plugin to convert charts into Kustomize compatible packages
 - [helm-cos](https://github.com/imroc/helm-cos) - Plugin to manage repositories on Tencent Cloud Object Storage
 - [helm-edit](https://github.com/mstrzele/helm-edit) - Plugin for editing release's values
 - [helm-env](https://github.com/adamreese/helm-env) - Plugin to show current environment
-- [helm-export](https://github.com/maorfr/helm-export) - Plugin to export releases from namespace to a file (disaster recovery)
 - [helm-gcs](https://github.com/nouney/helm-gcs) - Plugin to manage repositories on Google Cloud Storage
 - [helm-github](https://github.com/sagansystems/helm-github) - Plugin to install Helm Charts from Github repositories
 - [helm-hashtag](https://github.com/balboah/helm-hashtag) - Plugin for tracking docker tag hash digests as values
-- [helm-import](https://github.com/maorfr/helm-import) - Plugin for importing releases from (exported) file to a namespace (disaster recovery)
 - [helm-inject](https://github.com/maorfr/helm-inject) - Plugin for injecting additional configurations during release upgrade
 - [helm-k8comp](https://github.com/cststack/k8comp) - Plugin to create Helm Charts from hiera using k8comp
 - [helm-last](https://github.com/adamreese/helm-last) - Plugin to show the latest release
 - [helm-local](https://github.com/adamreese/helm-local) - Plugin to run Tiller as a local daemon
 - [helm-logs](https://github.com/maorfr/helm-logs) - Plugin to view changed releases over time
 - [helm-monitor](https://github.com/ContainerSolutions/helm-monitor) - Plugin to monitor a release and rollback based on Prometheus/ElasticSearch query
 - [helm-nuke](https://github.com/adamreese/helm-nuke) - Plugin to destroy all releases
-- [helm-restore](https://github.com/maorfr/helm-restore) - Plugin to restore the last deployed release to its original state
+- [helm-restore](https://github.com/maorfr/helm-restore) - Plugin to restore a deployed release to its original state
 - [helm-secrets](https://github.com/futuresimple/helm-secrets) - Plugin to manage and store secrets safely
 - [helm-stop](https://github.com/IBM/helm-stop) - Plugin for stopping a release pods
 - [helm-template](https://github.com/technosophos/helm-template) - Debug/render templates client-side
 - [helm-tiller](https://github.com/adamreese/helm-tiller) - Additional commands to work with Tiller
 - [helm-unittest](https://github.com/lrills/helm-unittest) - Plugin for unit testing chart locally with YAML
 - [Tillerless Helm v2](https://github.com/rimusz/helm-tiller) - Helm plugin for using Tiller locally and in CI/CD pipelines
 
+To help you get started, [helm-plugin-utils](https://github.com/maorfr/helm-plugin-utils) holds some utility functions to be used within Helm plugins.
```

Comment:
Can we please add this to the list instead? While it isn't technically a plugin, it is a third-party utility outside of the helm org for plugins, and I want to avoid providing "ad space" in the documentation wherever possible so we can keep the documentation unbiased. Thanks!

```
- [helm-plugin-utils](https://github.com/maorfr/helm-plugin-utils) - utility functions to be used within Helm plugins.
```

Reply:
was wondering if that is appropriate ;)
fixed according to your review.